### PR TITLE
[Added] Chinese

### PR DESCRIPTION
**Pull request by heng1025**
add Chinese translation

### CODE REVIEW

**Review by codemaster138 — APPROVED** [review]
I don't know Chinese, so I can't guarantee this translation is correct, but I can say this is a very solid PR!

/cc @jasonsaayman

**Comment by jasonsaayman** [comment]
@codemaster138 I am waiting for @chinesedfan to see if he can review since he is a native Chinese speaker, let's give him a couple day's else I will merge.

**Comment by codemaster138** [comment]
@jasonsaayman Has @chinesedfan taken a look at this yet?

**Comment by chinesedfan** [comment]
Looking. Will give feedbacks this weekend.

File: posts/zh/urlencoded.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+在浏览器中，可以使用[`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) API，如下所示：
+
+```js
+const params = new URLSearchParams();
+params.append('param1', 'value1');
+params.append('param2', 'value2');
+axios.post('/foo', params);
+```
+
+> 请注意，所有的浏览器(参见 [caniuse.com](http://www.caniuse.com/#feat=urlsearchparams))都不支持 `URLSearchParams` ，但是可以使用[polyfill](https://github.com/WebReflection/url-search-params) (确保 polyfill 全局环境)
```
Comment (chinesedfan) [inline posts/zh/urlencoded.md:22]:
```suggestion
> 请注意，不是所有的浏览器(参见 [caniuse.com](http://www.caniuse.com/#feat=urlsearchparams))都支持 `URLSearchParams` ，但是可以使用[polyfill](https://github.com/WebReflection/url-search-params) (确保 polyfill 全局环境)
```

File: posts/zh/handling_errors.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+```js
+axios.get('/user/12345', {
+  validateStatus: function (status) {
+    return status < 500; // 处理状态码小于500的情况
+  }
+})
+```
+
+使用 `toJSON` 可以获取更多关于HTTP的错误信息。
```
Comment (chinesedfan) [inline posts/zh/handling_errors.md:40]:
```suggestion
使用 `toJSON` 可以获取更多关于HTTP错误的信息。
```

File: posts/zh/api_intro.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+##### axios(url[, config])
+
+```js
+// 发起一个 GET 请求 (默认请求方式)
+axios('/user/12345');
+```
+
+### 请求方式别名
+
+为了方便起，已经为所有支持的请求方法提供了别名。
```
Comment (chinesedfan) [inline posts/zh/api_intro.md:47]:
```suggestion
为了方便起见，已经为所有支持的请求方法提供了别名。
```

**Review by chinesedfan — CHANGES_REQUESTED** [review]
Added some comments. By the way, shall we maintain those /public html files in git? Looks like auto-generated.

**Comment by codemaster138** [comment]
> By the way, shall we maintain those /public html files in git? Looks like auto-generated.

@chinesedfan No, those public files don't need to be in git. The only reason I didn't add them to the .gitignore is that I thought we'd be hosting with GitHub pages, but since we're not, you can delete them.

**Comment by heng1025** [comment]
> > By the way, shall we maintain those /public html files in git? Looks like auto-generated.
> 
> @chinesedfan No, those public files don't need to be in git. The only reason I didn't add them to the .gitignore is that I thought we'd be hosting with GitHub pages, but since we're not, you can delete them.

Github Action also can achieve same goal, even if there were not /public html files.

**Review by chinesedfan — APPROVED** [review]

**Comment by codemaster138** [comment]
> > > By the way, shall we maintain those /public html files in git? Looks like auto-generated.
> > 
> > 
> > @chinesedfan No, those public files don't need to be in git. The only reason I didn't add them to the .gitignore is that I thought we'd be hosting with GitHub pages, but since we're not, you can delete them.
> 
> Github Action also can achieve the same goal, even if there were not /public html files.

Yeah, I know, I guess I just wasn't thinking very hard 😅

**Comment by jasonsaayman** [comment]
@codemaster138 please open a new pull request that will handle the public HTML files, for now, I will merge this.

**Pull request merged**